packages/sign/src/identity/ci.ts

```diff
@@ -38,9 +37,9 @@ export class CIContextProvider implements IdentityProvider {
   // Invoke all registered ProviderFuncs and return the value of whichever one
   // resolves first.
   public async getToken() {
-    return promise
-      .promiseAny(providers.map((getToken) => getToken(this.audience)))
-      .catch(() => Promise.reject('CI: no tokens available'));
+    return Promise.any(
+      providers.map((getToken) => getToken(this.audience))
+    ).catch(() => Promise.reject('CI: no tokens available'));
```

Comment:
Now that we're on Node 16 we can swap our custom `promiseAny` function for the `Promise.any` implementation in the standard library.